source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -78,6 +79,24 @@ class ACRDateField: NSView {
         testButton.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
     }
     
+    private func setupTrackingArea() {
+        let trackingArea = NSTrackingArea(rect: bounds, options: [.activeAlways, .inVisibleRect, .mouseEnteredAndExited], owner: self, userInfo: nil)
+        addTrackingArea(trackingArea)
+    }
+    
+    override func mouseEntered(with event: NSEvent) {
+        guard let contentView = event.trackingArea?.owner as? ACRDateField else { return }
+        if #available(OSX 10.14, *) {
```

Comment:
I think this if else check can be done in a util method as its being done everywhere. 
And in all the inputs we can just call 
`contentView.textField.backgroundColor = ColorUtils.getHoverColor()`

And utils has a method - 
```
func hoverColor = if OSX 10.14 
unemphasizedSelectedTextBackgroundColor
 else 
 windowBackgroundColor
 ```
 Something similar

Reply:
Got it Manu
Will do this